Menu/main_menu.gd

```diff
@@ -7,5 +7,5 @@ func _ready():
 
```

Comment:
(I would mark the line but I can't do it from the review screen).
In the `_ready()` function, we can load the home scene.
Something like this:
```
# Load the home screen
$content_switcher.load_scene_home("res://Menu/SubMenu/Home/home.tscn")
```

Next, we can **remove the home scene from the main menu scene in the editor** since this line loads it instead. The content switcher needs to load its scenes via code, rather than in the editor. (maybe there is a way to do it in the editor, but I don't know how to without listing every possible scene that uses it in content switcher.)